Sources/UI/Components/TextInputView/TextInputView.swift

```diff
@@ -101,6 +101,9 @@ extension TextInputView {
     }
 
     private func updateAppearance(animated: Bool) {
```

Comment:
Isn't it better to remove `animated` argument. And add `- TODO: ` note to set it later when we migrate to `UIStackView`? If I correctly understand the change.

```suggestion
    // - TODO: return `animated` argument when we resolve issues with UICollectionView animation (see: https://github.com/optile/checkout-ios/pull/154#pullrequestreview-807716808)
    private func updateAppearance() {
```

Reply:
I see two options here:

1. We keep the current code as intact as possible while disabling animations (the way it is now)
2. We remove all animation code as if it never existed in the first place

I chose 1 because I think this code is really valuable, I really want to have this enabled again and the UI working really well as it should, and also I don't know when we'll be able to get it done and it'd be great if when the time comes we can just change these 2 small things and get the original functionality back.

@siebeprojects @Sorix I'm looking forward to hearing your opinions on this.

Reply:
@caiomello when the TextInputView has the option to either enable/disable animations then we should use that functionality ourselves to disable animations if we do not want it at the moment.
Either we set the Default value to "false", or we initialise all TextInputViews with the animation flag to false.

Reply:
You are absolutely correct @siebeprojects! That's definitely the way to go. The component provides the functionality, it's up to the example app to decide if it animates or not of course. I don't know how that didn't occur to me 🤦🏻‍♂️